### ISSUE TITLE
Attempting to use search_by_plaintext causes a syntax error in Postgres

### ISSUE THREAD

**Issue by campbecf** [jmazzi/crypt_keeper#103]
```
  PG::SyntaxError: ERROR:  syntax error at or near "AS"
       LINE 1: ...."last_four" = $1 AND ((pgp_sym_decrypt(cast(full AS bytea),...
```

Wouldn't this require a Create Cast statement to work?

**Comment by campbecf** [comment]
This was due to using a key word in Postgres. https://www.postgresql.org/docs/9.6/static/sql-keywords-appendix.html

#137 should resolve it.

**Comment by jmazzi** [comment]
Closed by #137
